scripts/lint-multi-repo.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+//internal
+const Logger = require('../packages/logger/lib/index');
+const eslintConfig = require('../packages/eslint-config/lib/index');
+
+//external
+const { glob } = require('glob');
+const { ESLint } = require('eslint');
+const fs = require('fs');
+
+(async () => {
+	try {
+		// create an instance that extends the dotcom-reliability-kit/eslint-config via the imported module
+		const eslint = new ESLint({
+			useEslintrc: false,
+			// @ts-ignore >> not able to fix this and doesn't stop the script from running fine
+			overrideConfig: eslintConfig
```

Comment:
OK maybe try this?

```suggestion
			overrideConfig: eslintConfig,
			useEslintrc: false,
```

Reply:
But that's what I thought by assigning this `useEslintrc` property to `false` meant it was not using the native eslint files 🤔 . Seems you feel by changing the order this should work now?

Reply:
Ignore me entirely 😂 I somehow missed that you were already setting this property